### DISCUSSION TITLE
Clean up all old packages

### DISCUSSION THREAD

**Discussion by ericnantel** [ScoopInstaller/Scoop#6197] [Ideas]
I recently found out about why I was missing 10s of Gb and it was all those previously installed versions of packages.
So I looked for a way to cleanup all of them, but it seems that the command : 'scoop cleanup' requires a package name.
So it would be great to make that command work similarily like scoop update.

By the way, I love this package manager.

Eric.

**Comment by chawyehsu** [comment]
`scoop cleanup -a -k` does the job. `scoop help cleanup` shows you the help page.
